File: scratch/get_user.js
const sqlite3 = require('sqlite3').verbose();
const db = new sqlite3.Database('pb_data/data.db');

db.all("SELECT id, email, role FROM users LIMIT 5", (err, rows) => {
  if (err) return console.error(err);
  console.log("Users:", rows);
});

File: scratch/get_user.py
import sqlite3

conn = sqlite3.connect('pb_data/data.db')
cursor = conn.cursor()

cursor.execute("SELECT id, email, role FROM users LIMIT 5")
users = cursor.fetchall()
print("=== USERS ===")
for u in users:
    print(u)

conn.close()
